geometry_msgs/Point position
std_msgs/Float64 angle
---
std_msgs/Bool success
std_msgs/Float64[] joint_positions